test/fixtures/cookbooks/yum_test/metadata.rb

```diff
@@ -1,8 +1,3 @@
 name 'yum_test'
```

Comment:
If we are in here anyway can we rename this folder and this cookbook to be `test` in line with our normal conventions please?